tests/FINDOLOGIC/Tests/PropertyTest.php

```diff
@@ -27,4 +27,28 @@ public function testAddingMultipleValuesWithoutUsergroupCausesException()
         $property->addValue('foobar');
         $property->addValue('foobar');
     }
+
+    /**
+     * @expectedException \FINDOLOGIC\Export\Data\PropertyKeyNotAllowedException
+     */
+    public function testReservedPropertyKeyOrdernumberCausesException()
+    {
+        $property = new Property('ordernumber');
+    }
+
+    /**
+     * @expectedException \FINDOLOGIC\Export\Data\PropertyKeyNotAllowedException
+     */
+    public function testReservedPropertyKeyThumbnailCausesException()
```

Comment:
An opposite test would also be interesting to confirm that this exception *only* occurs for reserved property keys, and no others.

Reply:
Done, used much of your code from above